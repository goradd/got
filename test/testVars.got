package testOut

import (
	"bytes"
	"fmt"
)

func givesString() string {
	return "Me"
}

func givesInt() int {
	return -5
}

func givesUint() uint {
	return 6
}

func TestVars(buf *bytes.Buffer) {
{{= "Evaluates to a string" }}

{{
Here is a number: {{i givesInt() }}
And another: {{u givesUint() }}
And a float: {{f 45/6 }}
Stringer: {{v "something" }}
}}

{{# Here is a comment }}

}


