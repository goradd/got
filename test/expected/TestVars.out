Evaluates to a string
Here is a number: -5
And another: 6
And a float: 7
Stringer: something
    Num = 0
    Num = 1
    Num = 2
    Num = 3
    Num = 4